### DISCUSSION TITLE
Can I require a file being in the REPL?

### DISCUSSION THREAD

**Discussion by Chemaclass** [phel-lang/phel-lang#176] [Q&A] [answered]
I find the REPL really good, and I was wondering if we can already import/require other modules being inside the repl

> vendor/bin/phel repl

Is this possible or we need to implement it, @jenshaase? 
If yes, please give me an example. If no, then I will create an issue for it.

**Comment by jenshaase** [comment]
Currently, you just can use the `ns` function. But there should be some repl utils to make this work. I made an issue for that: #179 179

**Reply by Chemaclass** [reply]
Perfect. I can help you with this one. If you want, write down in the ticket what do you have in mind for "some repl utils" and assign it to me.

**Comment by Chemaclass** [comment, marked as answer]
This was solved here https://github.com/phel-lang/phel-lang/issues/179 by being able to `require` and `use` external namespaces/objects.
Check the docs: https://phel-lang.org/documentation/repl/
